### PR TITLE
Add a new csharp file

### LINKED ISSUE #9155: missing documentation on how to add "cs files" to a dotnet core project from visual studio code
I wanted to use visual studio code and dotnet core to write a console application, however, the tutorial section for using visual studio code with dotnet core doesn't have any explaination on how to have a project with more than one cs file.  so I gave up on on using .netcore and switched to python instead.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8cc2e9c2-f333-f6a1-064e-790d950b684c
* Version Independent ID: 25a4c9b9-2808-e296-df03-5837000dcbad
* Content: [Get started with C# and Visual Studio Code - C# Guide](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio-code)
* Content Source: [docs/core/tutorials/with-visual-studio-code.md](https://github.com/dotnet/docs/blob/master/docs/core/tutorials/with-visual-studio-code.md)
* Product: **dotnet-core**
* GitHub Login: @kendrahavens
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by kendrahavens**
## Summary

Add a section on how to add a new .cs file to a project. If approved, we should add the same to the VB doc.

Fixes #9155

### Reviewers
@KathleenDollard

### CODE REVIEW

File: docs/core/tutorials/with-visual-studio-code.md [hunk 1]
```diff
@@ -82,6 +82,51 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
...
+2. Name your file `Class1.cs`. You must save it with a `.cs` extension at the end for it to be recognized as a csharp file.
+3. Add the code below to create your first class. Make sure to include the correct namespace so you can reference it from your `Program.cs` file.
+``` csharp
+using System;
+
+namespace HelloWorld
+{
+    public class Class1
+    {
+        public Class1(){}
```
Comment (BillWagner) [inline docs/core/tutorials/with-visual-studio-code.md]: nit: I'd remove this line as unnecessary.

**Review by BillWagner — APPROVED** [review]
This is a great addition @kendrahavens 

I had one suggestion, if you wanted to update the code before you :shipit:. Overall, LGTM.

File: docs/core/tutorials/with-visual-studio-code.md [hunk 2]
```diff
@@ -82,6 +82,51 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
...
+using System;
+
+namespace HelloWorld
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Class1 c1 = new Class1();
+            Console.WriteLine("Hello World!" + c1.ReturnMessage());
```
Comment (pkulikov) [inline docs/core/tutorials/with-visual-studio-code.md]: nit: doc slowly move to using string interpolation (then no need in the white space in the `ReturnMessage` method)
```suggestion
            Console.WriteLine($"Hello World! {c1.ReturnMessage()}");
```

**Comment by kendrahavens** [comment]
@BillWagner or @mairaw  I don't have permissions to merge. Thank you for reviewing!

File: docs/core/tutorials/with-visual-studio-code.md [hunk 3]
```diff
@@ -82,6 +82,50 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 > [!TIP]
 > For more information and troubleshooting tips on .NET Core debugging with OmniSharp in Visual Studio Code, see [Instructions for setting up the .NET Core debugger](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger.md).
 
+## Add a Class
```
Comment (mairaw) [inline docs/core/tutorials/with-visual-studio-code.md]: Sentence case for headings

```suggestion
## Add a class
```

I can merge for you and add VB later.

File: docs/core/tutorials/with-visual-studio-code.md [hunk 4]
```diff
@@ -82,6 +82,50 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 > [!TIP]
```
Comment (mairaw) [inline docs/core/tutorials/with-visual-studio-code.md]: also ms.date on line 6 should be updated.

File: docs/core/tutorials/with-visual-studio-code.md (hunk 3 above)
Comment (kendrahavens) [reply docs/core/tutorials/with-visual-studio-code.md]: Yay! Thank you!

**Comment by kendrahavens** [comment]
Thank you all!!

**Pull request merged**